test/lib/util.js

```diff
@@ -240,6 +240,44 @@ async function trackStarted(track) {
   await new Promise(resolve => track.once('started', resolve));
 }
 
+const activeIceCandidatePairStatsNullProps = {
+  chrome: new Set([
+    'availableIncomingBitrate',
```

Comment:
Hmm, I think this is only `null` in our tests because we don't wait long enough? Or maybe because we aren't using the new stats API?

Reply:
@markandrus Usually the behavior is if the property exists, it will be set to 0 if incoming media is not received yet, I think.